.github/CONTRIBUTING.md

```diff
@@ -30,15 +30,18 @@ Our procedures for contributing bigger changes, code in particular, generally fo
 
 *  For user-facing changes, add a bullet to the top of `NEWS.md` (i.e. just below the first header). Follow the style described in <https://style.tidyverse.org/news.html>.
 
+*  If you're changing the scripts in `data-raw/`, you can run tests locally: `devtools::test()` (because the
```

Comment:
This is the only real change to `CONTRIBUTING.md`. The rest of the diff is a pre-commit check the removes trailing whitespace.